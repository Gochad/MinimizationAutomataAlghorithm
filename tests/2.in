A B C D E F G H
A 
C
0 1
A 1 F
A 0 B
B 0 G
B 1 C
C 0 A
C 1 C
D 0 C
D 1 G
E 0 H
E 1 F
F 0 C
F 1 G
G 0 G
G 1 E
H 0 G
H 1 C