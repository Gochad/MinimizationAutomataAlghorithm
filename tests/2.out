G AE BH DF C 
A 
C 
0 1 
G 0 G
G 1 AE
AE 0 BH
AE 1 DF
BH 0 G
BH 1 C
DF 0 C
DF 1 G
C 0 AE
C 1 C
